e2e/linodeobjectstoragebucket-controller/minimal-linodeobjectstoragebucket/03-assert.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
 kind: LinodeObjectStorageBucket
 metadata:
-  name: linodeobjectstoragebucket-sample
+  name: (join('-', [$namespace, 'backups']))
```

Comment:
instead of computing this multiple times i would create a test level binding with the computed name

Reply:
is `$namespace` available at the top binding level? I am seeing this error
```
=== NAME  chainsaw
    | 19:35:38 | chainsaw | @main | INTERNAL  | ERROR |
        === ERROR
        <nil>: Internal error: variable not defined: $namespace
```
with this binding def
```
spec:
  template: true
  bindings:
    - name: bucketName
      value: (join('-', [$namespace, 'backups']))
```

Reply:
You're right, it's a bug. Fixing it, it will be in the next release.

Reply:
https://github.com/kyverno/chainsaw/pull/1097